resources/views/components/tables/headers/mobile/encapsulated/address.blade.php

```diff
@@ -1,7 +1,12 @@
+@props([
+    'model',
+    'withUsername' => false,
```

Comment:
```suggestion
    'withoutUsername' => true,
```
maybe for consistency with the other components?

Reply:
can do yeh. i generally try to do flag props where they just need passing (e.g. `<x-componentx with-username />`) but for consistency I'll change it :oleglala: